src/web/app/components/question-text-with-info/question-text-with-info.component.html

```diff
@@ -1,14 +1,14 @@
 <span class="d-inline-flex align-items-center">
-  <span *ngIf="shouldShowDownloadQuestionResult" class="col-auto">
+  <h3 *ngIf="shouldShowDownloadQuestionResult" class="col-auto question-number">
```

Comment:
In terms of headings order I believe the `h3`s should be `h2`s. Switching should cause minimal changes since we are already overriding the CSS anyway.